packages/sfui/frameworks/react/hooks/useScrollable/useScrollable.ts

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useRef, useState } from 'react';
+import { type UseScrollableOptions, Scrollable } from '@storefront-ui/react';
```

Comment:
Is it correct import for `Scrollable` class? Shouldn't be imported from `@storefront-ui/shared`?